q2_feature_classifier/custom.py

```diff
@@ -15,7 +15,11 @@
 
 class LowMemoryMultinomialNB(MultinomialNB):
     def __init__(self, alpha=1.0, fit_prior=True, class_prior=None,
-                 chunk_size=-1):
+                 chunk_size=0):
```

Comment:
OK we'll revert to keep this parameter consistent with the others. We were trying to avoid having two sentinels (-1 and 0, which both do the same thing) because it's likely to raise questions from users and we really only need one sentinel. But it's probably worth tackling this later so all parameters get the same updates.

For now, it'd be helpful to document this behavior in the parameter help text whenever that gets added. Thanks!